templates/openstack/consulting.html

```diff
@@ -1,389 +1,493 @@
-        <p>Once you are certain OpenStack is the right choice we can deliver your cloud for you. Only then do you need to commit the devops talent required to run your cloud, or <a href="/openstack/managed">outsource that to Canonical</a> as well.</p>
-      </div>
-      <div class="col-4 u-hide--small u-vertically-center">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/f5d56df2-OpenStack-consulting+design.svg",
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <div class="p-logo-section">
+      <p class="p-logo-section__title u-align--center p-muted-heading">Deployed openstack with Canonical
+      </p>
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/d0b7aa28-logo-deutsche-telekom.png",
             alt="",
-            height="155",
-            width="240",
+            width="160",
+            height="80",
             hi_def=True,
-            loading="lazy",
-          ) | safe
-        }}
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5d97cdd1-Tesco_Logo.svg",
+            alt="",
+            width="132",
+            height="36",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/baeb8320-ntt-logo.svg",
+            alt="",
+            width="130",
+            height="47",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/b41ec3f5-SBI-Bits_logo.png",
+            alt="",
+            width="615",
+            height="187",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5af38159-etisalat_logo.svg",
+            alt="",
+            width="108",
+            height="41",
+            hi_def=True,
+            loading="sudlazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f7651dc9-Logo-GARR-300dpi.png",
+            alt="",
+            width="2417",
+            height="917",
+            hi_def=True,
+            loading="lazy"
```

Comment:
These logos should have the `class="p-logo-section__logo"` to follow the [Vanilla pattern](https://vanillaframework.io/docs/patterns/logo-section). You will find the image module with added class round the site.